### DISCUSSION TITLE
Cannot use `for k, v in X` with "hashmap"-like data structures

### DISCUSSION THREAD

**Discussion by alpharder** [mitsuhiko/minijinja#231] [Q&A] [answered]
## Description

`Error: cannot unpack: not a sequence` while trying to iterate over with JSON-object hash-map alike values.

## Reproduction steps
1. At playground (WASM):
<img width="1279" alt="Screenshot 2023-03-24 at 04 38 03" src="https://user-images.githubusercontent.com/1678161/227380520-cbf26d6f-898c-48f1-95c1-0bfa0c94566c.png">
2. I initially reproduced this locally when passing `serde_json::Value` based  as template rendering context:
<img width="630" alt="Screenshot 2023-03-24 at 04 43 15" src="https://user-images.githubusercontent.com/1678161/227380872-d5dabcf3-df75-4532-a622-d57526e55cb3.png">

Additional helpful information:

- Version of minijinja: `0.30.7`
- Version of rustc: `rustc 1.67.1 (d5a82bbd2 2023-02-07)`
- Operating system and version: MacOS

## What did you expect
I expect this to iterate over key-value pairs

**Comment by mitsuhiko** [comment, marked as answer]
To iterate over key value pairs you need to use `|items` or `|dictsort`:

```jinja
{% for key, value in my_dict|items %}
  {{ key }}: {{ value }}
{% endfor %}
```

Or alternatively

```jinja
{% for key in my_dict %}
  {{ key }}: {{ my_dict[key] }}
{% endfor %}
```
